spring-xd-dirt/src/main/resources/application.yml

```diff
@@ -179,8 +179,11 @@ xd:
         # Streams
         - GET    /streams/definitions            => hasRole('ROLE_VIEW')
         - DELETE /streams/definitions            => hasRole('ROLE_CREATE')
+        - GET    /streams/definitions.*          => hasRole('ROLE_VIEW')
```

Comment:
Would be better to have the dot version just below the non-dot, can fix on merge